src/utils/__tests__/paths.spec.ts

```diff
@@ -46,6 +61,10 @@ describe('paths', () => {
       win32:  '/usr/local/bin',
       darwin: '/usr/local/bin',
     },
+    resources: {
+      win32:  RESOURCES_PATH,
+      darwin: RESOURCES_PATH,
```

Comment:
Hmm. It's more interesting to test the packaged case (because that has the more unusual path on macOS); however, testing that doesn't appear to be easy, so we can think about that later.